--- config/sources/families/bcm2711.conf	2025-05-27 12:22:16
+++ config/sources/families/bcm2711.conf-mod	2025-05-27 12:26:07
@@ -342,12 +342,13 @@
 		kernel=vmlinuz
 		initramfs initrd.img followkernel
 		arm_64bit=1
+		dtoverlay=dwc2,dr_mode=peripheral
 	EOD
 }
 
 function pre_umount_final_image__write_raspi_cmdline() {
 	cat <<- EOD > "${MOUNT}"/boot/firmware/cmdline.txt
-		console=serial0,115200 console=tty1 loglevel=1 root=LABEL=${ROOT_FS_LABEL} rootfstype=${ROOTFS_TYPE} fsck.repair=yes rootwait logo.nologo cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory
+		console=tty1 modules-load=dwc2,g_cdc cfg80211.ieee80211_regdom=GB console=ttyGS0,115200 loglevel=1 root=LABEL=${ROOT_FS_LABEL} rootfstype=${ROOTFS_TYPE} fsck.repair=yes rootwait logo.nologo cgroup_enable=cpuset cgroup_memory=1 cgroup_enable=memory
 	EOD
 
 	if [[ "${SHOW_DEBUG}" == "yes" ]]; then
